src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -145,7 +145,10 @@ def end_title
 
       def device_value(device)
         if device.is?(:blk_filesystem)
-          device.type.to_human_string
+          device_name =  [device.type.to_human_string]
+          device_name << device.blk_device_basename if device.multidevice?
+
+          device_name.join(" ")
```

Comment:
The order with translations is always tricky. I would protect us against it:

```suggestion
          if device.multidevice?
            format(_("%{type} %{device}"),
              type: device.type.to_human_string,
              device: device.blk_device_basename)
            else
              device.type.to_human_string
            end
```